build-system/externs/amp.extern.js

```diff
@@ -533,13 +533,15 @@ Activity.prototype.getIncrementalEngagedTime = function(name, reset) {};
 AccessService.prototype.getAccessReaderId = function() {};
 AccessService.prototype.getAuthdataField = function(field) {};
 // Same for amp-analytics
+
 /**
  * The "get CID" parameters.
  * - createCookieIfNotPresent: Whether CID is allowed to create a cookie when.
  *   Default value is `false`.
  * @typedef {{
  *   scope: string,
  *   createCookieIfNotPresent: (boolean|undefined),
+ *   cookieName: (string|undefined),
```

Comment:
Nice catch. This PR is already showing its worth.

Reply:
Thanks :).  I only caught this due to the rule against duplicate typedefs, but unfortunately I wasn't able to figure out how to keep it. Because while linting is run against all of the files at the same time, Closure Compiler evaluates many of the externs in isolation.

Reply:
I actually don't know why GetCidDef is here 
@dvoytenko @erwinmombay do you recall? Is the cid API being called externally?

Reply:
@laanka is that a question we can reasonably defer to the future or a separate issue? (It seems unrelated to the purpose of this PR which is to make externs `prettier`)